docs/how-to-generate-diagrams.md

```diff
@@ -0,0 +1,9 @@
+#How to generate diagrams
+
+Proxy diagrams should be generated by preference with Plantuml.
```

Comment:
According to their website, they seem to prefer the name to be capitalised as "plantUML".